src/detours.cpp

```diff
@@ -1436,8 +1447,8 @@ static PDETOUR_TRAMPOLINE detour_alloc_trampoline(PBYTE pbTarget)
 
     // We need to allocate a new region.
 
-    // Round pbTarget down to 64KB block.
-    pbTarget = pbTarget - (PtrToUlong(pbTarget) & 0xffff);
+    // Round pbTarget down to 64KB block.
+    pbTarget = pbTarget - (ULONG)((ULONG_PTR)pbTarget & 0xffff);
```

Comment:
This should not be achieving anything really.
PtrToUlong was invented to encapsulate this pattern, except you moved the parenthesis around.